forte/base_classes/active_space_solver.h

```diff
@@ -130,6 +131,12 @@ class ActiveSpaceSolver {
     /// Set if read wave function from file as initial guess
     void set_read_initial_guess(bool read_guess) { read_initial_guess_ = read_guess; }
 
+    /// Return the coupling coefficients for a given state
+    CICouplingCoefficients coupling_coefficients(const StateInfo& state, int max_level = 2) const;
```

Comment:
Because `ActiveSpaceSolver` has the information of determinant space. For example, CASCI and ACI hold different determinants.

The coupling coefficients can be formed between any two determinants. But we just care about the active space part, i.e., a CAS, RAS, ACI, etc. So, if we ever have a CISD class, the CISD class can also output a `CICouplingCoefficients`.

Reply:
I wrote that when I didn't realize that there was a sum over determinants. I meant to delete this, apologies.